app/templates/organen/orgaan/mandatarissen.hbs

```diff
@@ -25,13 +25,21 @@
 </AuToolbar>
 
 <div class="au-o-box">
-  <Organen::MandatarisTable
+  <Organen::DraftMandatarisList
+    @mandatarissen={{@model.mandatarissen}}
     @content={{@model.mandatarissen}}
     @sort={{this.sort}}
     @page={{this.page}}
     @size={{this.size}}
-    @personRoute="mandatarissen.persoon"
-    @mandaatRoute="mandatarissen.mandataris"
+    @title=""
+    @showVoornaam={{true}}
+    @showNaam={{true}}
+    @showFractie={{true}}
+    @showFunctie={{true}}
+    @showRangorde={{false}}
+    @showBevoegdheid={{false}}
+    @showStartMandaat={{true}}
+    @showEindeMandaat={{true}}
```

Comment:
I think it is nice have settings for which column you show, but I think some colums should always be shown, no?

Reply:
Yes indeed! I was thinking at a 'showAll' & 'showPersonInfo' & de andere dan apart ?

Reply:
Why: When looking at the mockups the dates or not in there and the person info is always visible